scio-core/src/main/scala/com/spotify/scio/values/SideInput.scala

```diff
@@ -42,6 +42,42 @@ trait SideInput[T] extends Serializable {
   private[values] val view: PCollectionView[_]
 }
 
+object SideInput {
+
+  /**
+   * Wrap a view of a Singleton as a [[SideInput]]. In most cases you want to use
```

Comment:
`singleton`, `multi-map` as they're not types, but `List`, `Iterable`, `Map` as they are proper types?